resspect/fit_lightcurves.py

```diff
@@ -163,6 +175,33 @@ def __init__(self):
         self.sncode = 0
         self.sntype = ' '
 
+    def _iterate_snpcc_lc_data(
+            self, lc_data: np.ndarray) -> Tuple[np.ndarray, list]:
+        photometry_raw = []
+        header = []
+        for each_row in lc_data:
+            name = each_row[0]
+            value = each_row[1]
+            if name == 'SNID:':
+                self.id = int(value)
+                self.id_name = 'SNID'
+            elif name == 'SNTYPE:':
+                self.sample = 'test' if value == '-9' else 'train'
```

Comment:
can this `-9` value be pulled into some constants file somewhere rather than hardcoded here?